getssl

```diff
@@ -3162,12 +3164,12 @@ debug "created SAN list = $SANLIST"
 if [[ "$DUAL_RSA_ECDSA" == "false" ]] && [[ -s "$DOMAIN_DIR/${DOMAIN}.key" ]]; then
   case "${PRIVATE_KEY_ALG}" in
     rsa)
-      if grep --silent -- "-----BEGIN EC PRIVATE KEY-----" "$DOMAIN_DIR/${DOMAIN}.key"; then
```

Comment:
I used --silent intentionally as some time ago I had to learn that (contrary to the man page -q/--quiet is not exactly the same as --silent. To completely mute grep I had to use --silent. Unfortunately I cannot recall what environment this was. So be prepared for surprises. (Maybe it's a better solution not to use any option but to discard any output with >/dev/null 2>&1)